pyccel/codegen/pipeline.py

```diff
@@ -196,12 +212,67 @@ def handle_error(stage):
 
         #------------------------------------------------------
         # TODO: collect dependencies and proceed recursively
-    #    if recursive:
-    #        for dep in parser.sons:
-    #            # Call same function on 'dep'
-    #            pass
+        # if recursive:
+        #     for dep in parser.sons:
+        #         # Call same function on 'dep'
+        #         pass
         #------------------------------------------------------
 
+        # Determine if the module or program requires internal libraries
+        # by iterating over internal_libs list that containes internal libs
+        # implemented inside pyccel and compare them by module includes
+        # returned by printer includes through get_additional_imports function
```

Comment:
I got lost while reading this sentence, which is 4 lines long and does not contain any commas... Could you rephrase it please?

Typo: containes -> contains
Language: compare them by -> compare them with

Reply:
I think the comment is too long.
what do you think about writing this comment instead :
```
# Iterate over the internal_libs list and determine if the printer
# requires an internal lib to be included.
```